ruby/vampires.rb

```diff
@@ -0,0 +1,75 @@
+# method to determine the likelihood of being a vampire
+def determine_vampire (likes_garlic, needs_insurance, age_check, name)
+  result = "Results inconclusive."
+
+  if age_check && (likes_garlic || needs_insurance)
+    result = "Probably not a vampire."
+  end
+  if !age_check && (!likes_garlic || !needs_insurance)
+    result = "Probably a vampire."
+  end
+  if !age_check && !likes_garlic && !needs_insurance
+    result = "Almost certainly a vampire."
+  end
+  if name == "Drake Cula" || name == "Tu Fang"
+    result = "Definitely a vampire."
+  end
+  return result
```

Comment:
No need to return result, in fact we could omit this line entirely as the last evaluted expression in a method is the return of that method.